backend/db/database.go

```diff
@@ -26,11 +26,11 @@ type Database interface {
 
 func NewDatabaseFromEnv() (Database, error) {
 	if connection == nil {
-		runningEnv, wasSet := os.LookupEnv("RUN_ENV")
+		runningEnv, wasSet := os.LookupEnv("DB_TYPE")
```

Comment:
Just to be consistent
```suggestion
		dbType, wasSet := os.LookupEnv("DB_TYPE")
```